src/core_plugins/kibana/public/discover/components/field_chooser/field_chooser.html

```diff
@@ -3,7 +3,7 @@
     <ui-select
       class="index-pattern-selection"
       ng-model="selectedIndexPattern"
-      on-select="setIndexPattern($item)"
+      on-select="setField('index', $item)"
```

Comment:
I think this instance of `setIndexPattern` should probably stay.

Reply:
Derp! Good catch.